go/test/endtoend/messaging/main_test.go

```diff
@@ -41,28 +41,24 @@ var (
 	userKeyspace         = "user"
 	lookupKeyspace       = "lookup"
 	createShardedMessage = `create table sharded_message(
-		time_scheduled bigint,
 		id bigint,
-		time_next bigint,
+		time_next bigint default 0,
 		epoch bigint,
```

Comment:
Why not default epoch too?

Reply:
This is to demonstrate that it works with no default.